dts/arm/nrf52840_pca10056.dts

```diff
@@ -42,19 +42,19 @@
 
 		boot_partition: partition@0 {
 			label = "mcuboot";
-			reg = <0x000000000 0x00008000>;
+			reg = <0x000000000 0x0000C000>;
 		};
 		slot0_partition: partition@8000 {
 			label = "image-0";
-			reg = <0x00008000 0x00006c000>;
+			reg = <0x0000C000 0x000069000>;
 		};
 		slot1_partition: partition@74000 {
 			label = "image-1";
-			reg = <0x00074000 0x00006c000>;
+			reg = <0x00075000 0x000069000>;
 		};
 		scratch_partition: partition@e0000 {
 			label = "image-scratch";
-			reg = <0x000e0000 0x0001d000>;
+			reg = <0x000de000 0x0001e000>;
 		};
 
 		/*
```

Comment:
The comment and partition below here need to be fixed too (unused portion starts at 0xfc000, not 0xfd000)